third-party-major.json

```diff
@@ -83,7 +94,7 @@
       "minimumReleaseAge": "14 days"
     },
     {
-      "excludePackagePatterns": ["^seek-jobs/", "^seek-oss/"],
+      "matchPackageNames": ["!/^seek-jobs//", "!/^seek-oss//"],
```

Comment:
Do the slashes here need to be escaped?

https://docs.renovatebot.com/string-pattern-matching/#use-regex101-to-test-your-patterns

Reply:
Renovate migrate did that 😅